kyaml/openapi/Makefile

```diff
@@ -5,7 +5,6 @@ MYGOBIN = $(shell go env GOBIN)
 ifeq ($(MYGOBIN),)
 MYGOBIN = $(shell go env GOPATH)/bin
 endif
-API_VERSION := "v1.21.2"
```

Comment:
Why don't we want to keep this to provide the default/current value? I think `make all` is broken because of this removal. Also, it's still referenced in the readme.

Reply:
You're right, it makes sense to keep. Added it back